homeassistant/components/binary_sensor/__init__.py

```diff
@@ -75,6 +75,9 @@
 # On means problem detected, Off means no problem (OK)
 DEVICE_CLASS_PROBLEM = "problem"
 
+# On means yes, Off means no
+DEVICE_CLASS_QUESTION = "question"
```

Comment:
Changing an entity model needs to be approved first in an issue in our architecture repo.

https://developers.home-assistant.io/docs/en/entity_index.html#changing-the-entity-model

Reply:
Missed that. Thank you. I’ll start there.